i18n/fr/docusaurus-plugin-content-docs/version-22.10/monitoring/basic-objects/hosts.md

```diff
@@ -13,127 +13,133 @@ Pour créer un hôte manuellement :
+* **Période de contrôle** : définit la période temporelle durant laquelle l’ordonnanceur vérifie activement le statut de l’objet.
+* **Nombre de contrôles avant validation de l'état** : définit le nombre de contrôles à effectuer avant de confirmer le statut de l’hôte comme non-OK.
   lorsque le statut est validé, le processus de notification est enclenché.
-* Le champ **Intervalle normal de contrôle** est exprimé en minutes. Il définit l’intervalle entre chaque vérification lorsque
+* **Intervalle normal de contrôle** : exprimé en minutes. Il définit l’intervalle entre chaque contrôle lorsque
   le statut de l’hôte est OK.
-* Le champ **Intervalle non-régulier de contrôle** est exprimé en minutes. Il définit l’intervalle de validation du statut non-OK de l’hôte.
-* Les champs **Contrôle actif activé** et **Contrôle passif activé** activent / désactivent les contrôles actifs et passifs.
+* **Intervalle non-régulier de contrôle** : exprimé en minutes, désigne l'intervalle de temps entre chaque contrôle réalisé afin de confirmer l'état non-OK de l'hôte. Une fois le nombre de contrôles de validation réalisé, l'intervalle entre deux contrôles revient à son rythme normal.
+* **Contrôle actif activé** et **Contrôle passif activé** activent / désactivent les contrôles actifs et passifs. [Les contrôles passifs](../../monitoring/passive-monitoring/enable-snmp-traps.md) sont les informations que la ressource supervisée envoie au moteur de supervision sans que celles-ci aient été activement demandées.
 
 ## Onglet Notification
+Jetez un oeil à notre documentation sur les [notifications](../../alerts-notifications/notif-concept.md) et les [contacts](contacts.md) pour en savoir plus sur ces sujets.
 
-* Le champ **Notification activée** permet d’activer ou de désactiver les notifications concernant l’objet.
-* La liste **Contacts liés** indique les contacts qui recevront les notifications.
-* La liste **Groupes de contacts liés** tous les contacts appartenant aux groupes de contacts définis recevront les
-  notifications.
+* **Notification activée** : activer ou désactiver les notifications concernant l’objet.
+* **Contacts liés** : contacts qui recevront les notifications. Ces contacts doivent être d'abord configurés à la page **Configuration > Utilisateurs**.
+* **Groupes de contacts liés** : tous les contacts appartenant aux groupes de contacts définis recevront les
+  notifications. Les groupes doivent d'abord être configurés à la page **Configuration > Utilisateurs**.
   
-  Si à la page **Administration > Paramètres > Centreon web**, l'option **Vertical inheritance only** est activée, deux cases supplémentaires apparaissent :
+  **Vertical inheritance only** : établit les contacts et/ou groupes de contacts à notifier. Elle se trouve dans l'onglet **Administration > Paramètres > Centreon web**. Une fois activée, deux cases supplémentaires apparaissent :
```

Comment:
This version is different from EN version becasue both options are missing.
Moreover, **Vertical inheritance only** doesn't appears in the form. You must indicate that if **Vertical inheritance only** is defined in **Administration > Paramètres > Centreon web** so 2 options appear:
* Contacts hérités additionnels
* Groupes de contacts hérités additionnels